src/hrflow_connectors/core/auth.py

```diff
@@ -52,3 +52,24 @@ def update(
             headers.update(
                 {"Authorization": "OAuth {}".format(self.get_access_token())}
             )
+
+
+class SmartToken(Auth):
+    """
+    Authorization to access and use smartrecruiters static token
+    """
+
+    access_token: str
+
+    def get_access_token(self):
+        return self.access_token
+
+    def update(
+        self,
+        params: Union[Dict[str, str], None] = None,
+        headers: Union[Dict[str, str], None] = None,
+        payload: Union[Dict[str, str], None] = None,
+        cookies: Union[Dict[str, str], None] = None,
+    ):
+        if headers is not None:
+            headers.update({"X-SmartToken": "{}".format(self.get_access_token())})
```

Comment:
Don't forget to test this class like I did with `Outh2PasswordCredentialsBody`. 

See how I did it here `tests/core/test_auth.py` and add your test in the same file.